src/components/shared/eventcard/eventcard.css.js

```diff
@@ -0,0 +1,94 @@
+import styled from 'styled-components';
+import { grid, space, color, typography, layout } from 'styled-system';
+import Text from 'components/shared/text';
+import Box from 'components/shared/box';
+
+// style-system default breakpoints
+// const breakpoints = ['40em', '52em', '64em']
+export const Article = styled.article`
+  display: grid;
+  grid-template-columns: auto 1fr;
+  grid-template-rows: auto auto 1fr;
+  grid-template-areas:
+    'date title'
+    'image image'
+    'rsvp rsvp';
+
+  @media screen and (min-width: 64em) {
+    grid-template-columns: auto 1fr;
+    grid-template-rows: auto auto 1fr;
+    grid-template-areas:
+      'date title'
+      'blank image'
+      'blank rsvp';
+  }
+
+  background-color: white;
+  box-sizing: border-box;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+`;
+
+export const Description = styled.p`
+  display: block;
+  ${space};
+  ${color};
+  ${typography};
+`;
+
+const RSVPBox = styled(Box)(space, grid, layout);
+
+RSVPBox.defaultProps = {
+  gridArea: 'rsvp',
+  display: 'flex',
+  alignItems: 'flex-start',
+  flexDirection: ['column', 'row'],
+  justifyContent: 'space-between',
+};
+
+export const IconList = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: start;
+`;
+
+export const IconRow = styled.div`
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  ${space};
+`;
+
+const IconText = styled(Text)(color, space, typography);
+const IconLink = styled(Text)(color, space, typography);
+
+IconText.defaultProps = {
+  color: 'Grays.100',
+  fontSize: 3,
+  ml: 3,
+};
+
+IconLink.defaultProps = {
+  color: 'Grays.100',
+  textDecoration: 'underline',
+  fontSize: 3,
+  ml: 3,
+};
+
+export const ImageWrapper = styled.div`
+  position: relative;
+  display: block;
+  grid-area: image;
+  padding-bottom: 56%;
```

Comment:
What's 56% represent here? In general I think percentage based sizing is a code smell... it's almost never what you want and isn't great for responsivity.

Reply:
Trying to maintain a rough 16:9 ratio on the event images. 

Bulma uses 56.25% for 16x9.
![image](https://user-images.githubusercontent.com/4359586/69504190-913e3580-0eee-11ea-91ec-921109534765.png)

I've seen Adam Wathan use this approach in a recent Tailwind screencast as well: https://tailwindcss.com/course/locking-images-to-a-fixed-aspect-ratio

I checked this at various breakpoints and it seemed to look alright. If there's a better way to do this let me know.

Reply:
Ah, I see what the intention here is now. :+1: LGTM